integration/push_test.go

```diff
@@ -165,11 +165,11 @@ func TestLegacyOCI(t *testing.T) {
 
 			imageName := ubuntuImage
 			copyImage(sh, dockerhub(imageName), regConfig.mirror(imageName))
+
 			var buildOpts []indexBuildOption
-			if tc.supportLegacyRegistry {
-				buildOpts = append(buildOpts, withLegacyRegistrySupport)
+			if !tc.supportLegacyRegistry {
```

Comment:
can we invert `supportLegacyRegistry` and change to `tc.supportArtifactRegistry`. Since `supportLegacyRegistry` no longer exists.